### DISCUSSION TITLE
DNS_PROBE_FINISHED_NXDOMAIN for first setup

### DISCUSSION THREAD

**Discussion by Tritfire** [Authenticator-Extension/Authenticator#947] [Q&A] [answered]
Hello ! I have an issue for the first setup of one of my computers, it runs on opera (which I never had any issues with it before), after the installation of the extension we get automatically redirected to that url : https://otp.ee/quickstart which doesn't work because of some dns issues...

![image_2022-09-13_190331058](https://user-images.githubusercontent.com/7926315/189963013-f0dd2268-a1b4-4b7a-828e-0559543f12df.png)

I already tried chrome or changing my dns settings and enabling dns over https...

Any ideas please ?

Thanks :)

**Comment by Sneezry** [comment, marked as answer]
There's a DNS issue with the domain. It should work in minutes.

**Reply by Tritfire** [reply]
Yeap perfect thanks ;)
